test/e2e/catalog_e2e_test.go

```diff
@@ -78,9 +79,18 @@ var _ = Describe("Catalog represents a store of bundles which OLM can use to ins
 			},
 		}
 
-		crd := newCRD(genName("ins"))
+		crd := newCRD(genName("ins-"))
 		csv := newCSV(packageStable, ns.GetName(), "", semver.MustParse("0.1.0"), []apiextensions.CustomResourceDefinition{crd}, nil, nil)
 
+		defer func() {
+			Eventually(func() error {
+				return ctx.Ctx().KubeClient().ApiextensionsInterface().ApiextensionsV1().CustomResourceDefinitions().Delete(context.TODO(), crd.GetName(), metav1.DeleteOptions{})
+			}).Should(Or(Succeed(), WithTransform(k8serror.IsNotFound, BeTrue())))
+			Eventually(func() error {
+				return ctx.Ctx().Client().Delete(context.TODO(), &csv)
```

Comment:
WDYT about `return client.IgnoreNotFound(ctx.Ctx().Client().Delete(context.TODO(), &csv))` instead so we don't need any (somewhat) complex gomega matcher expressions?

Reply:
I had no idea that existed! I'll make the changes

Reply:
Done!